view/rooms/app.js

```diff
@@ -754,6 +754,26 @@ function setupSocketIOClient(bots) {
 
   socket.on('disconnect', () => {
     debugMessage('Socket Disconnected');
+    const elem = document.createElement('div');
+    elem.id = 'snackbar';
+    elem.className = 'slds-notify slds-notify_toast slds-theme_offline show';
+    elem.innerHTML = '<div class="snackContent">You have ' +
+      'lost connection with Refocus, so bot data and bot actions ' +
+      'maybe unreliable. Please save any unfinished work and ' +
+      '<a href="javascript:window.location.reload(true)">' +
+      'click here to reconnect</a></div>';
```

Comment:
the link should just be the word `here` (not the whole phrase) OR get rid of the word "here" and just say "... and click reconnect to continue." where "reconnect" is the link.